test/behaviour/concept/thing/attribute/BUILD

```diff
@@ -18,34 +18,49 @@
 #
 
 package(default_visibility = ["//test/behaviour:__subpackages__"])
+
 load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
-load("@graknlabs_common//test/server:rules.bzl", "grakn_java_test")
+load("//test/behaviour:rules.bzl", "grakn_core_and_cluster_behaviour_test")
 
 java_library(
     name = "steps",
     srcs = [
-        "AttributeSteps.java"
+        "AttributeSteps.java",
     ],
+    visibility = ["//visibility:public"],
     deps = [
         # Internal Package Dependencies
         "//:client-java",
         "//test/behaviour/concept/thing:steps",
-        "//test/behaviour/util:util",
-        "//test/behaviour/connection:steps",
+        "//test/behaviour/util",
+        "//test/behaviour/connection:steps-base",
 
         # External Maven Dependencies
         "@maven//:junit_junit",
         "@maven//:io_cucumber_cucumber_java",
     ],
-    visibility = ["//visibility:public"],
 )
-
-grakn_java_test(
+grakn_core_and_cluster_behaviour_test(
     name = "test",
     srcs = [
-        "AttributeTest.java"
+        "AttributeTest.java",
     ],
     test_class = "grakn.client.test.behaviour.concept.thing.attribute.AttributeTest",
+    data = [
+        "@graknlabs_behaviour//concept/thing:attribute.feature",
+    ],
+    native_grakn_artifact_core = "//test:native-grakn-core-artifact",
+    native_grakn_artifact_cluster = "//test:native-grakn-cluster-artifact",
+    connection_steps_core = "//test/behaviour/connection:steps-core",
+    connection_steps_cluster = "//test/behaviour/connection:steps-cluster",
```

Comment:
It's a little bit arbitrary that the SetUp steps are in `ConnectionSteps`, but given how silly our solution for setup and teardown is anyway, I think we can safely overlook this.

Reply:
yeah blood has been shed here